cursor.md

```
@@ -0,0 +1,32 @@
+
+使用方法：修改需求里的内容，将需求和步骤内容作为指令让cursor进行执行。
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)